libraries/adsk/adsklib/colorcorrect.mtlx

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <nodedef name="ND_colorcorrect" node="colorcorrect1" nodegroup="adjustment">
```

Comment:
`nodegroup` is being used as single level "classification" for now. Would like to extend this to allow sub-classification like /shader/wood/walnut, or /procedural/image/triplanarProjection/ etc. TBD.